modules/kronatools/ktimporttaxonomy/main.nf

```diff
@@ -0,0 +1,39 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process KRONATOOLS_KTIMPORTTAXONOMY {
+    tag "${meta.classifier}-${meta.id}"
```

Comment:
What's the `classifier` attribute? 
If this is only needed for the tag, I would rather not use it at all as not all users might have it in their `meta`.

Reply:
As far as I can tell, the way that the[ current ](https://github.com/nf-core/mag/blob/d4f00bc2d178c1c8dc25b576c9e77e70511d5d21/workflows/mag.nf#L332)`nf-core/mag` pipeline is written, this process is run separately for results produced by `kraken2` and `centrifuge`, which are two metagonic classifcation engines referred to by `classifier`

Reply:
You are right, nf-core/mag uses this process for Kraken2 and Centrifuge output. However, I agree, `meta.classifier` is not appropriate for a generalized module. Local modules like the one you linked can do such informative steps, but nf-core modules probably should not, generalization is more important, i.e. minimize required input.